fixtures/valid-questionnaire-template.js

```diff
@@ -2206,6 +2206,15 @@ const validQTemplate = {
                 }
             ]
         }
+    },
+    taxonomies: {
```

Comment:
could you make it something like:

````
taxonomies: {
    'some-category': {
         taxa: {
             'some-theme': {
                 title: 'Some theme title'   
    }
}
````
This would make it a bit clearer for what the "customisable/changeable" things in this object are.